pkg/apis/options/options.go

```diff
@@ -22,6 +22,7 @@ type Options struct {
 	ProxyPrefix        string   `flag:"proxy-prefix" cfg:"proxy_prefix"`
 	PingPath           string   `flag:"ping-path" cfg:"ping_path"`
 	PingUserAgent      string   `flag:"ping-user-agent" cfg:"ping_user_agent"`
+	MetricsPort        string   `flag:"metrics-port" cfg:"metrics_port"`
```

Comment:
Once we get to the finish line on the code content aspect of this PR, we should be sure to get these new flags into the documentation. Prometheus support and which metrics we are exposing likely deserve there own documentation page explaining everything too.

Reply:
Added the flags to the docs. Might not be all the right places so I am happy to add more if you could point me to the right place.